lua/hawtkeys/init.lua

```diff
@@ -68,6 +68,8 @@ local defaultConfig = {
     lhsBlacklist = { "<plug>", "Þ" },
 }
 
+local auGroup = vim.api.nvim_create_augroup("hawtkeys", { clear = false })
```

Comment:
```suggestion
local auGroup = vim.api.nvim_create_augroup("hawtkeys", { clear = true })
```

We do want to clear it - that way, if the plugin is reloaded we avoid duplicate autocmds.